### PR TITLE
Production Pull

### PR DESCRIPTION
**Pull request by bubby932**
Increases global rate limit by 2.5x and adds more restrictive rate limits for image fetching and other sensitive operations.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ app.set('trust proxy', 1);
 
 var GlobalLimiter = RateLimit({
     windowMs: 1*60*1000,
-    max: 100,
+    max: 250,
```
Comment (bubby932) [inline index.js:20]: Simple requests can have their rate limit upped since they're not too sensitive.

File: routers/rooms.js [hunk 2]
```diff
@@ -71,8 +72,13 @@ router.route("/room/:room_id/info")
...
     });
 
+const download_limit = rateLimit({
+    'windowMs': 60 * 10 * 1000,
+    'max': 10
+});
+
 router.route("/room/:room_id/subrooms/:subroom_id/versions/:version_id/download")
-    .get(authenticateToken, async (req, res) => {
+    .get(authenticateToken, download_limit, async (req, res) => {
```
Comment (bubby932) [inline routers/rooms.js:81]: Downloading rooms now has a rate limit due to the fact that it reads from Firebase, which has a bandwidth limit.

File: routers/img.js [hunk 3]
```diff
@@ -58,14 +58,21 @@ const uploadRateLimit = rateLimit({
...
+
 // 1 hour cache
 const imgCache = new NodeCache({
     "deleteOnExpire": true,
     "stdTTL": 60 * 60
 });
 
 router.post("/upload", uploadRateLimit, middleware.authenticateToken, async (req, res) => {
-    if(config.disable_image_upload && !req.user.developer) return res.status(409).send({"message": "Access denied - image upload have been disabled by the system administrator.", "code": "uploads_disabled"});
+    if(config.disable_image_upload && !req.user.developer) return res.status(409).send({"message": "Access denied - image uploads have been disabled by the system administrator.", "code": "uploads_disabled"});
```
Comment (bubby932) [inline routers/img.js:75]: Phrasing

File: routers/img.js [hunk 4]
```diff
@@ -210,7 +217,7 @@ router.get("/:id/info", async (req, res) => {
     }
 });
 
-router.get("/:id", async (req, res) => {
+router.get("/:id", fetch_rate_limit, async (req, res) => {
```
Comment (bubby932) [inline routers/img.js:220]: Added a simple rate limit to image fetching to slow down the usage of Firebase bandwidth.

**Review by bubby932 — COMMENTED** [review]
Please review when possible.

**Comment by bubby932** [comment]
@Bobrobot1 you will review NOW!!!

**Review by Bobrobot1 — APPROVED** [review]

**Comment by bubby932** [comment]
no way

**Pull request merged**